extensions/noteplan-3/src/lib/note-utilities.ts

```diff
@@ -14,6 +15,10 @@ export enum NoteType {
   Project = "project",
 }
 
+interface Preferences {
```

Comment:
Let's extract the preferences to another file and only expose a getter function, i.e
`export const getPreferences = () => getPreferenceValues<Preferences>();`

Reply:
Yeah, that makes sense.